aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -132,8 +137,9 @@ module AptosFramework::Stake {
         max_lockup_duration_secs: u64,
         // Whether validators are allow to join/leave post genesis.
         allow_validator_set_change: bool,
-        // The maximum rewards (as a % of staked amount) given out every epoch.
-        rewards_rate_percentage: u64,
+        // The maximum rewards given out every epoch. This will be divided by the rewards rate denominator.
+        // For example,
+        rewards_rate: u64,
```

Comment:
why not have both a numerator and denominator as fields in this struct?

Reply:
Good idea! Updated.